src/Tizen.NUI/src/public/BaseComponents/FlexContainer.cs

```diff
@@ -46,7 +47,7 @@ public class FlexContainer : View
             Tizen.NUI.Object.GetProperty(flexContainer.swigCPtr, FlexContainer.Property.CONTENT_DIRECTION).Get(out temp);
             return (ContentDirectionType)temp;
         });
-        /// This will be public opened in tizen_5.0 after ACR done. Before ACR, need to be hidden as inhouse API.
+        /// <summary> Property of ContentDirection </summary>
```

Comment:
is it "Property of FlexDirection" ? could you check and fix it? ^^; Thank you~

Reply:
Oh Yes, you're right. Sorry for my mistake :(
Thanks for letting me know!